include/aspect/postprocess/visualization.h

```diff
@@ -562,6 +563,16 @@ namespace aspect
                                  const std::string &solution_file_prefix,
                                  const std::vector<std::string> &filenames);
 
+
+        /**
+         * A function, called from the execute() function, that takes a
+         * DataOut object, does some preliminary work with it, and then
+         * writes the result out to files via the writer() function (in the
+         * case of VTU output) or through the XDMF facilities.
+         */
```

Comment:
please document the return value

Reply:
I've already got follow-up patches in the queue so have put the documentation into the following commits. I hope that's ok with you.